content/en/docs/reference/command-line-tools-reference/feature-gates/informer_metrics.md

```diff
@@ -0,0 +1,13 @@
+---
+title: InformerMetrics
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Enable support for exposing informer/reflector metrics.
```

Comment:
My suggestion: add _informer_ and _reflector_ into the [Kubernetes glossary](https://kubernetes.io/docs/reference/glossary/), and then refer to those using [glossary shortcodes](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#glossary).

**Not needed for the alpha**